test/system/505-networking-pasta.bats

```diff
@@ -340,6 +341,12 @@ function pasta_test_do() {
            "Container route not matching host"
 }
 
+# FIXME: this test does not run parallel:
+#    Error: pasta failed with exit code 1:
+#    inotify_init1(): Too many open files, use a timer
+#    Couldn't set IPv6 route(s) in guest: Invalid argument
+#    [11:40:08.368457588] [ rc=126 (** EXPECTED 0 **) ]
```

Comment:
Does this actually only fail when run in parallel? I don't see why this would fail at all like that.

I don't have a ipv6 setup so it works for me locally of course.

However I also got another strange failure:
```
 ✗ [505] Podman unshare --rootless-netns with Pasta
   tags: para
   (from function `bail-now' in file test/system/helpers.bash, line 187,
    from function `die' in file test/system/helpers.bash, line 898,
    from function `run_podman' in file test/system/helpers.bash, line 500,
    in test file test/system/505-networking-pasta.bats, line 792)
     `CONTAINERS_HELPER_BINARY_DIR="$PODMAN_TMPDIR" run_podman 125 unshare --rootless-netns ip addr' failed
   
   [18:51:43.767063241] $ /home/pholzing/go/src/github.com/containers/podman/bin/podman info --format {{.Host.Pasta.Executable}}
   [18:51:43.979041761] /usr/bin/pasta
   Object "show" is unknown, try "ip help".
   
   [18:51:44.025423256] $ /home/pholzing/go/src/github.com/containers/podman/bin/podman unshare --rootless-netns ip addr
   [18:51:44.076929321] 1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
       link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
       inet 127.0.0.1/8 scope host lo
          valid_lft forever preferred_lft forever
       inet6 ::1/128 scope host proto kernel_lo
          valid_lft forever preferred_lft forever
   2: enp9s0u2u1u2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 65520 qdisc fq_codel state UNKNOWN group default qlen 1000
       link/ether 32:3d:45:a3:ca:c8 brd ff:ff:ff:ff:ff:ff
       inet 192.168.188.25/24 brd 192.168.188.255 scope global noprefixroute enp9s0u2u1u2
          valid_lft forever preferred_lft forever
       inet6 fe80::303d:45ff:fea3:cac8/64 scope link proto kernel_ll
          valid_lft forever preferred_lft forever
   #/vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
   #| FAIL: exit code is 0; expected 125
   #\^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

The only way I can see that failing is if the symlink is removed before we run podman there.
`ln -s /usr/bin/false $PODMAN_TMPDIR/pasta`

Reply:
>Does this actually only fail when run in parallel? I don't see why this would fail at all like that.

Actually it fails 100% of the time on my laptop, parallel or not. I only just realized that this morning but haven't taken the time to pursue it.

The other failure... is it possible that there's a parallel test somewhere that is reconfiguring (globally) podman to not use or invoke pasta? Maybe somehow it's reverting to the old networking code?

Reply:
I am running `hack/bats --rootless --tag=para 505` and I don't think there is anything there sets slirp4netns

Reply:
Yes, I see the timeout one frequently on my laptop runs. Have not seen it in CI. Also see:
```
# [11:46:41.000112389] $ bin/podman unshare --rootless-netns ip addr
# [11:46:41.916046191] Error: rootless netns: write ref counter: open /run/user/14904/containers/networks/rootless-netns/ref-count: no such file or directory
# [11:46:41.919143909] [ rc=127 (** EXPECTED 0 **) ]
```
and
```
# [11:46:42.061015059] $ bin/podman run --rm --network=pasta:-a,169.254.0.2,-g,169.254.0.1,-n,24 quay.io/libpod/testimage:20240123 grep host.containers.internal /etc/hosts
# [11:46:42.180804548] Error: pasta failed with exit code 1:
# Couldn't set IPv6 route(s) in guest: Invalid argument
# [11:46:42.183910374] [ rc=126 (** EXPECTED 0 **) ]
```
...neither one of which has shown up in CI yet. I don't have the skills to dive into those.

Reply:
but `Podman unshare --rootless-netns with Pasta` seems consistent locally even in non parallel runs so likely just a red hearing as well

Reply:
yeah ok my local setup got in a borken state somehow the rootless-netns ref-counter got of sync

And now I realize yes this test cannot be run in parallel `Podman unshare --rootless-netns with Pasta` because if the rootless-netns is already setup then pasta is not called (false) so not error is returned.

I suggest to move this test to 500-networking for now